### PR TITLE
Created CCP2, Subscription2 and Authorization2. contracts

### PR DESCRIPTION
**Pull request by MubarakOke**

### CODE REVIEW

File: frontend-material/txt.txt
Comment (VictoriaAde) [inline frontend-material/txt.txt:1]: @MubarakOke The frontend is still using the previously verified address that you removed. 
Please add them back, you can add comments to differentiate them from the new addresses - v1 or v2.

**Review by VictoriaAde — CHANGES_REQUESTED** [review]

File: src/DAO.sol
Comment (VictoriaAde) [inline src/DAO.sol:1]: @Adecom16 We are now making use of CCP2, please make use of that instead.

**Review by VictoriaAde — CHANGES_REQUESTED** [review]

File: src/DAO.sol
Comment (Adecom16) [reply src/DAO.sol:1]: alright

**Comment by VictoriaAde** [comment]
I am closing this PR because a lot things aren't right.

@MubarakOke  @Adecom16.
Please make the PR again, this time in separate branches.
Please take note of the addresses, we are still using the addresses for the CCP1 version.
Right now, we should have two versions of addresses, CCP1 and CCP2, take note of that.

**Pull request closed without merging**